src/video/wayland/SDL_waylandopengles.c

```diff
@@ -41,6 +41,8 @@ int Wayland_GLES_LoadLibrary(SDL_VideoDevice *_this, const char *path)
 
     ret = SDL_EGL_LoadLibrary(_this, path, (NativeDisplayType)data->display, _this->gl_config.egl_platform);
 
+    _this->gl_config.swap_behavior_preserved_bit = 1;
```

Comment:
This bit is only available if the EGL version >= 1.4, so the current version should be checked before enabling it.